src/docs/index.jsx

```diff
@@ -253,6 +259,61 @@ const Component = React.createClass({
 						})}
 					</tbody>
 				</table>
+				{!_.isEmpty(childComponents) ? (
+					<section>
+						<h3>Child Components</h3>
+						{_.map(childComponents, (childComponent) => (
+							<section key={childComponent.displayName}>
+								<h4>{childComponent.displayName}</h4>
+								<div dangerouslySetInnerHTML={getDescriptionAsHtml(childComponent.description)} />
```

Comment:
Is this Div necessary? couldn't this prop be set on the above `<section>` instead?